src/components/SignerPanel/SigningStatus.js

```diff
@@ -110,64 +107,85 @@ class SigningStatus extends React.Component {
     )
   }
   render() {
-    const { status } = this.props
+    const { theme, status } = this.props
     return (
-      <React.Fragment>
-        <Status>
+      <div>
+        <Status
+          color={theme.feedbackContent}
+          background={theme.feedbackSurface}
+        >
           <StatusImage status={status} />
-          <p>{this.getLabel()}</p>
+          <p
+            css={`
+              margin-top: ${3.5 * GU}px;
+              ${textStyle('body2')};
+            `}
+          >
+            {this.getLabel()}
+          </p>
         </Status>
-        <AdditionalInfo>{this.getInfo()}</AdditionalInfo>
+        <Info>{this.getInfo()}</Info>
         {this.getCloseButton()}
-      </React.Fragment>
+      </div>
     )
   }
 }
 
 const Status = styled.div`
-  margin-top: 80px;
-  margin-bottom: 40px;
+  background: ${({ background }) => background};
+  height: 360px;
+  margin-bottom: ${3 * GU}px;
   display: flex;
   flex-direction: column;
   justify-content: center;
   align-items: center;
-  color: ${theme.textSecondary};
-  img {
-    margin-bottom: 20px;
-  }
+  color: ${({ color }) => color};
 `
 
-const AdditionalInfo = styled(Info)`
-  p + p {
-    margin-top: 10px;
-  }
-`
+const StatusImage = ({ status }) => {
+  const theme = useTheme()
+  const color = isSignatureError(status)
+    ? theme.negative
+    : isSignatureSuccess(status)
+    ? theme.positive
+    : theme.feedbackSurfaceContentSecondary
```

Comment:
@dizzypaty we don't have the exactly colour in the theme, but this is maybe the closest in intent?

This is what it looks like:

<img width="228" alt="Screen Shot 2019-08-14 at 6 13 50 PM" src="https://user-images.githubusercontent.com/4166642/63037380-52481f00-bebf-11e9-9b1d-ad7264522b5d.png">

Reply:
@AquiGorka The theme now has the colour in the design (`theme.hint`).